### ISSUE TITLE
Todos for repo cleanup

### ISSUE THREAD

**Issue by ashander** [ropensci/opendata#4]
Todo list (for myself) on cleanup 

minor
- [x] deal with ctv check error (see commit 87cf706)
- [x] cleanup unused subheads within "open (web-based) data available via packages" 

restructure
- [x] double check if anything in "open (web-based) data available via packages" belongs above in data-only or data discovery/archiving
- [x] mark wrappers for paid services with $ or similar (maybe add to separate top-level header)
- [x] convert content and makefile to markdown 

submission
- [ ] lastly, edits to make new `OpenData_2cran.ctv` and `OpenData_2cran.html` files if necessary

optional? @sckott what is this used for in webservices repo?
- [x] update `opendata.json` to reflect current contents

For completeness, the current task view top-level header structure (quoting @leeper from issue #205 in webservices): 
- data discovery (figshare, dvn, dryad, OAIHarvester, etc.) and data archiving (some of the same tools as the previous point)
- data-only packages (like the ones I cited previously)
- open (web-based) data available via packages (from, for example, rOpenSci, rOpenGov, rOpenHealth, etc.)

**Comment by sckott** [comment]
>  update opendata.json to reflect current contents

you can get rid of that - I was playing around with providing a json output of the entire task view for fun, but not needed

**Comment by ashander** [comment]
Updated todo list to reflect #16

**Comment by leeper** [comment]
I just converted [the contribution model for PackageDevelopment](https://github.com/leeper/PackageDevelopment/blob/master/CONTRIBUTING.md) to start by editing a simple markdown document, then the Makefile converts this to ctv, converts the ctv to html, and uses the html to build a README. Markdown seems to be a friendly interface for editing than a rough ctv file. Should we think about that model here, as well?

**Comment by ashander** [comment]
That looks like a good model. I edited the todo list above. 

But, not sure if it makes sense to do this as part of overall restructuring on that list, or just as a separate issue. I'm happy to include in with those tasks, but won't get to it till early next week.

**Comment by leeper** [comment]
I opened a separate issue for it. I'll deal with it later. https://github.com/ropensci/opendata/issues/17

**Comment by ashander** [comment]
As a related point, should we develop a timeline towards CRAN submission? 

If so, someone could add a milestone to mark issues necessary for submission.

@sckott thoughts?

**Comment by leeper** [comment]
I've modified the workflow (in #17) and updated the checklist. opendata.md is now the input file. OpenData.ctv is the output. I've also created a "Release to CRAN" milestone, with this to-do list as the only currently remaining issue.

**Comment by sckott** [comment]
@ashander I set the milestone as due for two weeks from now, change if needed

**Comment by ashander** [comment]
@sckott great. Aiming for a first pass later this week

**Comment by ashander** [comment]
Last todo after #27 is any by-hand edits necessary for cran submission. @sckott any pointer on cran task view submission? 

Also, seems like at the same time we should also update the webtech tv on CRAN

**Comment by sckott** [comment]
@ashander  You have to use subversion, so that's fun. I do:
- Whoever maintainer is has to have an account with R-forge https://r-forge.r-project.org/projects/ctv/
- Clone down the ctv repo from rforge - or update if you already have it
- add your completed `.ctv` file to the ctv repo in `ctv/pkg/inst/ctv`
- commit change with message

make sure `ctv::check_ctv_packages()` doesn't give any errors on the ctv file

Yeah, should update webtech too

**Comment by ashander** [comment]
Thanks! I will update later this week
On Wed, May 13, 2015 at 1:48 PM, Scott Chamberlain <notifications@github.com

**Comment by ashander** [comment]
Assuming these latest PRs are merged

**Comment by ashander** [comment]
Current status:
- all looks good
- registered with r-forge and requested access to ctv repo so I can add this

**Comment by sckott** [comment]
nice work!

**Comment by ashander** [comment]
Waiting to hear back on R-forge. @sckott do you know if only Admin (Achim Zeileis in this case) can add members?

**Comment by sckott** [comment]
Yes, I believe he's the only person, and things usually take a long time with task views

**Comment by leeper** [comment]
Nice work, @ashander!

**Comment by ashander** [comment]
Have just sent a direct email to the ctv maintainer (albeit at r-forge address). . .

**Comment by leeper** [comment]
@ashander @sckott Have we heard back from CRAN about getting this added as a TaskView?

**Comment by sckott** [comment]
I think @ashander was waiting to hear back?

**Comment by ashander** [comment]
Thanks for the push. Yes I heard back from Achim this summer. He added me
as a contributor to ctv, but he had some suggested changes that I said I'd
implement before creating it on CRAN. I haven't done them yet, but realize
I was remiss in not just bringing them to the group (been PhDing ....)

Below are Achim's suggestions. I could go either way on most, but don't
think any of us has time to e.g., contact maintainers of subject matter
views for each suggested subject matter package. OTOH, we could put
something in contributing guidelines and link to those subject-matter
views.

Let me know what you think (also happy to discuss offline)
- Jaime

~~~~ Suggestions from Achim ~~~
Generally, I think that splitting up the WebTechnologies view is a good
idea and also starting an OpenData view is a good idea.

However, the OpenData view is already huge and I think it would benefit
from a sharper focus. I would suggest to omit:

(1) Packages that would better fit into the subject-matter task views like
"Finance", "Econometrics", "Genetics", etc. If certain packages are missing
from these subject-matter task views, contact the corresponding maintainers
and suggest inclusion of the missing packages.

(2) Focus on packages that can access open online data resources - but omit
packages with static copies of openly available data. My feeling is that
there are far too many packages that qualify for the latter.

Furthermore, I would recommend to focus more on active CRAN packages. Of
course, it is ok to include one or the other important archived package but
the default is usually to exclude archived packages from the task views.
Also, do not include links to the Github repos of CRAN packages: If the
packages want to be found on Github (or whatever other repository), they
can include this in their own DESCRIPTION/URL. (Exceptions would be, if
there is particular added value on the Github project that is not on CRAN.)

On Fri, Oct 2, 2015 at 12:21 PM, Scott Chamberlain <notifications@github.com

> wrote:
> 
> I think @ashander https://github.com/ashander was waiting to hear back?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ropensci/opendata/issues/4#issuecomment-145130595.

**Comment by leeper** [comment]
I think we should go ahead with Achim's suggestions here. I think for our purposes (having something here on GitHub), it may make sense to list a lot of GitHub-only packages, but for the CRAN release let's do as is suggested.

We can also cut the data-only packages. How do we feel about creating a separate page for those that is only listed on GitHub?

**Comment by leeper** [comment]
I think we've addressed the major issues here, so I'm opening a new issue (#114) to discuss final changes before pushing to CRAN.
